src/wizard-settings/header-bidding-gam/index.js

```diff
@@ -68,151 +54,65 @@ const HeaderBiddingGAM = () => {
 		}
 		return data;
 	};
-	const updateOrderName = async () => {
-		setInFlight( true );
-		let data;
-		try {
-			data = await apiFetch( {
-				path: '/newspack-ads/v1/bidding/gam/order_count',
-				method: 'GET',
-			} );
-		} catch {
-			// Ignore errors.
-		} finally {
-			if ( data?.count ) {
-				setOrderName( `Newspack Header Bidding v${ data.count + 1 }` );
-			} else {
-				setOrderName( 'Newspack Header Bidding' );
-			}
-			setInFlight( false );
-		}
+	const getActiveOrders = () => {
+		return orders?.filter( order => order.is_archived === false ) || [];
```

Comment:
Non-blocking: should we sort this list alphabetically? It seems to appear in a somewhat random order, neither alphabetical nor by order of creation.

<img width="704" alt="Screen Shot 2022-02-18 at 1 54 19 PM" src="https://user-images.githubusercontent.com/2230142/154760170-2910f7dc-f658-491a-a00b-83598698c2b7.png">

Reply:
Good call! Done in cfa93b7ed35c033989406b995330a4ac05f728be